daemon/algod/api/algod.oas2.json

```diff
@@ -186,19 +186,19 @@
           "400": {
             "description": "Bad request",
             "schema": {
-              "$ref": "#/definitions/AccountsErrorResponse"
+              "$ref": "#/responses/AccountsErrorResponse"
             }
           },
           "401": {
             "description": "Invalid API Token",
             "schema": {
-              "$ref": "#/definitions/ErrorResponse"
+              "$ref": "#/responses/ErrorResponse"
             }
           },
           "500": {
             "description": "Internal Error",
             "schema": {
-              "$ref": "#/definitions/ErrorResponse"
+              "$ref": "#/responses/ErrorResponse"
```

Comment:
I think now that you have added "schema" inside the `ErrorResponse` type this should be:
```
"500": {
  "description": "Internal Error",
  "$ref": "#/responses/ErrorResponse"
}
```
so that you are not double-wrapping it inside two `"schema"` objects

Reply:
that is how the indexer spec does it https://github.com/algorand/indexer/blob/develop/api%2Findexer.oas2.json#L282-L284 and it would be good for the two to use the same style